utils/build/build.js

```diff
@@ -122,4 +123,11 @@ onChanges.push({
   script: 'utils/generate_types/index.js',
 });
 
+// Copy images.
+steps.push({
```

Comment:
You'd want to chokidar this in watch mode.

Reply:
I don't anticipate this changing live